framework/src/main/java/org/checkerframework/framework/util/typeinference/DefaultTypeArgumentInference.java

```diff
@@ -443,7 +443,7 @@ private Map<TypeVariable, AnnotatedTypeMirror> infer(
      * }</pre>
      *
      * TODO: NOTE WE ONLY DO THIS FOR InferredType results for now but we should probably include
-     * targest as well
+     * target as well
```

Comment:
What does `target` refer to here? Maybe it should have been `targets`?

Reply:
Done